File: task-03/C/Sub-task 3/sub-task-3.c
#include<stdio.h>
void main() {
     int n;
     printf("Enter the value of n: ");
     scanf("%d",&n);

     for(int i=0;i<n;i++){
	for(int j=i;j<n;j++){
	    printf("  ");
	}
	for(int j=0;j<=i;j++){
            printf("* ");
        }
	for(int j=0;j<i;j++){
	    printf("* ");
       	}
	printf("\n");
     }
     for(int i=0;i<=n;i++){
       	for(int j=0;j<i;j++){
            printf("  ");
        }
        for(int j=i;j<n;j++){
            printf("* ");
        }
        for(int j=i;j<=n;j++){
            printf("* ");
        }
        printf("\n");
      }
}

File: task-03/Ruby/Sub-task 3/sub-task-3.rb
puts "Enter the value of n: "
n = gets.chomp.to_i

for i in 0..n-1
    for j in i..n-1
        print "  "
    end	
    for j in 0..i
        print "* "
    end	
    for j in 0..i-1
        print "* "
    end
    puts(" ")
end
for i in 0..n
    for j in 0..i-1
        print "  "
    end
    for j in i..n-1
        print "* "
    end
    for j in i..n
        print "* "
    end
    puts(" ")
end
